app/views/users/new.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Users#new</h1>
+<p>Find me in app/views/users/new.html.erb</p>
```

Comment:
remove file that never used

Reply:
it will got error if i deleted this file.
![image](https://user-images.githubusercontent.com/68435978/91519923-83f32300-e926-11ea-91c3-5c47299e077f.png)

Reply:
Okie, sorry.
Because I dont see your UsersController, therefore I think this file is not used.
Keep this file, please

Reply:
Noted it. Thank you.